src/app/dashboard/dashboard.component.html

```diff
@@ -5,7 +5,7 @@
             <div>
                 <h4>Welcome</h4>
                 <p>Set up your Dashboard and start creating your awesome products!</p>
-                <a href="#">Create a New Project</a>
+                <a (click)="createProjectDialog.open()">Create a New Project</a>
```

Comment:
Don't use a tag if no need of href attribute. Because its behavior may change in different browser.